package.json

```diff
@@ -12,7 +12,7 @@
   "main": "main.js",
   "scripts": {
     "postinstall": "electron-builder install-app-deps && rimraf node_modules/dtrace-provider",
-    "test": "npm run eslint && npm run test-server && grunt test && npm run test-app && npm run test-modules",
+    "test": "yarn eslint && yarn test-server && grunt test && yarn test-app && yarn test-modules",
```

Comment:
I seem to recall running into some problems when we tried to move all of these sub-tasks to `yarn` like this. We should be careful here.

Reply:
I tried locally and all the tests seem to continue to pass as expected. You ok to give it a try and revert if we run into problems? Or let me know if you remember specific things to look out for.